main.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'lib/manipulation'
+
+class Main
+    file = Parser.new('games.log')
+    file.openfile
```

Comment:
Acho que poderia ter um nome mais descritivo pra essa função, porque tu não tá somente abrindo o arquivo. Tá abrindo o arquivo e retornando a primeira linha. Se for mudar, não esquecer de trocar na definição.

```suggestion
    file.get_first_line
```